todo/src/components/Todos.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { connect } from "react-redux";
+import {toggleToDo, removeToDo} from '../actions/actions';
+
+function Todos(props) {
+    console.log(props)
+    return(
+        <div className="todoList">
+        <h1>Todos:</h1>
+            <ul>
+
+                {props.todos.map( todo => {
+                    return(
+                <li key={todo.value} 
+                onClick={() => props.toggleToDo(todo.value)}
+                className={todo.completed ? 'done' : 'notDone'}
```

Comment:
This is an interesting approach! Why did you choose to return an li instead of a new separate 'todo' component? Most people made a separate `<Todo>` component, so I think it would be valuable for you to consider that possibility and why you chose your solution!